docs/guide/run-ansible.md

```diff
@@ -64,8 +64,8 @@ collecting the rewards.
 ::: tab 🪟 Window
 
 ```
-docker run -i --rm -v c:\zarb\:/zarb zarb/zarb key generate -p /zarb/keystore/validator_key.json
-docker run -i --rm -v c:\zarb\:/zarb zarb/zarb key generate -p /zarb/keystore/mintbase_key.json
+docker run -i --rm -v %USERPROFILE%\zarb:/zarb zarb/zarb key generate -p /zarb/keystore/validator_key.json
+docker run -i --rm -v %USERPROFILE%\zarb:/zarb zarb/zarb key generate -p /zarb/keystore/mintbase_key.json
 ```
 
 This command will generate new keys and save them at: `c:\zarb\keystore\`
```

Comment:
I am not a Windows user, but I think we can't spoon-feed them. This is an example to show how a key can be generated

Also, Look at line #71, it is still `c:\zarb\keystore\`. We have this location in other docs as well.

Reply:
sure but we ain't spoon-feeding them just providing working examples to try out